### PR TITLE
KFRD-25 Second part of integration tests was implemented

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
@placccebo

File: src/it/scala/com/bwsw/kafka/reader/KafkaReaderIntegrationTest.scala [hunk 1]
```diff
@@ -89,11 +92,79 @@ class KafkaReaderIntegrationTest extends fixture.FlatSpec {
...
+      groupId,
+      topicInfoList,
+      retrievedCount = sizeTestDataSet,
+      expectedCount = sizeTestDataSet
+    )
+
+    testForEmptyTopics(fixture.kafkaEndpoints, groupId, topicInfoList)
+  }
+
+  it should "process some part of events from single Kafka topic, save checkpoint data (indicating what events have been processed), " +
```
Comment (placccebo) [inline src/it/scala/com/bwsw/kafka/reader/KafkaReaderIntegrationTest.scala]: it should "process some part of events from single Kafka topic, save checkpoint data (indicating what events have been processed). Next, run from a scratch and repeat the two first steps for the rest of events. Then run again from a scratch and it should not receive any message"

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/it/scala/com/bwsw/kafka/reader/KafkaReaderIntegrationTest.scala [hunk 2]
```diff
@@ -65,11 +65,14 @@ class KafkaReaderIntegrationTest extends fixture.FlatSpec {
 
     fixture.producer.send(producerRecords.toList)
 
+    val sizeTestDataSet = testDataSet.size
```
Comment (placccebo) [inline src/it/scala/com/bwsw/kafka/reader/KafkaReaderIntegrationTest.scala]: testDataSetSize

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/it/scala/com/bwsw/kafka/reader/KafkaReaderIntegrationTest.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/kafka/reader/KafkaReaderIntegrationTest.scala]: ok

File: src/it/scala/com/bwsw/kafka/reader/KafkaReaderIntegrationTest.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/kafka/reader/KafkaReaderIntegrationTest.scala]: ok

**Comment by MedvedevBW** [comment]
@placccebo

**Review by placccebo — APPROVED** [review]

**Pull request merged**